rbvmomi.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.license  = 'MIT'
 
   spec.bindir = 'exe'
-  spec.files  = `git ls-files -z`.split("\x0").reject { |f| f.match(/^test\//) }
+  spec.files  = %w{LICENSE README.md} + Dir.glob("{lib,exe}/**/*")
```

Comment:
Sorry, for another request, but could you add the `vmodl.db` file in there as well. It's needed for rbvmomi to work.

Reply:
Added